docs/usage/java.md

```diff
@@ -46,6 +46,19 @@ Renovate does not support:
 - Catalogs with custom names that do not end in `.toml`
 - Catalogs outside the `gradle` folder whose names do not end in `.versions.toml` (unless overridden via [`fileMatch`](./configuration-options.md#filematch) configuration)
 
+### Gradle Plugin Support
+
+Renovate can also update [Gradle plugins](https://docs.gradle.org/current/userguide/plugins.html). It supports the
+`id(<pluginId>)` syntax as well as the `kotlin(<kotlinPluginId>)` shortcut for `id(org.jetbrains.kotlin.<kotlinPluginId>)`.
+
+For specifying `packageRules` it is important to know how `depName` and `packageName` are defined for a Gradle plugin:
+
+- The `depName` field is equal to `<pluginId>`
+- The `packageName` field is equal to `<pluginId>:<pluginId>.gradle.plugin`
+
+This is a direct consequence of the [Plugin Marker Artifact](https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers)
+naming convention.
```

Comment:
```suggestion
Renovate can also update [Gradle plugins](https://docs.gradle.org/current/userguide/plugins.html).
It supports the `id(<pluginId>)` syntax as well as the `kotlin(<kotlinPluginId>)` shortcut for `id(org.jetbrains.kotlin.<kotlinPluginId>)`.

For specifying `packageRules` it is important to know how `depName` and `packageName` are defined for a Gradle plugin:

- The `depName` field is equal to `<pluginId>`
- The `packageName` field is equal to `<pluginId>:<pluginId>.gradle.plugin`

This is a direct consequence of the [Plugin Marker Artifact](https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers) naming convention.
```

One sentence per line

Reply:
Sorry, I forgot to check against the style guide.